### PR TITLE
Update apps-update-experience.md

### PR DESCRIPTION
**Pull request by lambasu**
Updated the list of changes that trigger an upgrade request to the user

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/apps-update-experience.md [hunk 1]
```diff
@@ -27,11 +27,9 @@ In most cases, after app developers publish an app update, the new version autom
 * A bot was added or removed
 * An existing bot's "botId" property changed
 * An existing bot's "isNotificationOnly" property changed
-* The bot's "supportsFiles" property changed
-* A messaging extension was added or removed
+* A bot's SupportsCalling, SupportsVideo and SupportsFiles capability was added
```
Comment (JohanFreelancer9) [inline Teams/apps-update-experience.md:30]:
```suggestion
* A bot's SupportsCalling, SupportsVideo , and SupportsFiles capability was added
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lambasu, kindly commit the minor suggested change. Many thanks.

**Comment by lambasu** [comment]
committing

**Review by lambasu — COMMENTED** [review]
committing

File: Teams/apps-update-experience.md [hunk 2]
```diff
@@ -27,11 +27,9 @@ In most cases, after app developers publish an app update, the new version autom
 * A bot was added or removed
 * An existing bot's "botId" property changed
 * An existing bot's "isNotificationOnly" property changed
-* The bot's "supportsFiles" property changed
-* A messaging extension was added or removed
+* A bot's SupportsCalling, SupportsVideo , and SupportsFiles capability was added
```
Comment (JohanFreelancer9) [inline Teams/apps-update-experience.md:30]:
```suggestion
* A bot's SupportsCalling, SupportsVideo, and SupportsFiles capability was added
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @lambasu, kindly commit the suggested change again to remove the extra space. Many thanks.

**Review by lambasu — COMMENTED** [review]
committing

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@cichur PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**